content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -1,40 +1,45 @@
 # Data Versioning
 
-To **track** a large file or directory, place it in the <abbr>workspace</abbr>,
-and use `dvc add`:
+How cool would it be to make Git handle arbitraty large files and directories
+with the same performance as Git itself? Imagine you can do a `git clone` and
+see data files and ML model files in the workspace. Or do `git checkout` and
+switch to a different version of a 100Gb size file in a less than a second?
+
+The core part of DVC is a few commands that you can run along with Git to track
+a large file, ML model or a directory. To start tracking a file or directory,
+use `dvc add`:
 
 <details>
 
-### 👉 Expand to get an example dataset
+### ⚙️ Expand to get an example dataset
 
-Having [initialized](/doc/tutorials/get-started#initialize) a project, do this:
+Having initialized a project in the previous section, get the data file we will
```

Comment:
I feel like the link is more useful but no strong opinion.

Reply:
it makes it hard to support all these cross links - let's do them at the very end when it's more or less final